src/components/resultCard/ResultCard.jsx

```diff
@@ -0,0 +1,46 @@
+import { useQuiz } from "context";
+import styles from "./resultCard.module.css";
+
+function ResultCard({ questionObj, questionNumber }) {
+  const { question, options, answer } = questionObj;
+
+  const {
+    quizState: { questionAnswers },
+  } = useQuiz();
+
+  console.log(options, questionNumber);
+
+  return (
+    <>
+      <div className={styles.cardContainer}>
+        <h1 className="fs-m">{question}</h1>
+        <ul className={styles.list}>
+          {options.map((option) => {
+            console.log(
+              option,
+              answer,
+              questionNumber,
+              questionAnswers,
+              questionAnswers[questionNumber]
+            );
```

Comment:
Can remove this console log if not required

Reply:
Yeah, will remove!